packages/firestore/test/unit/specs/spec_test_runner.ts

```diff
@@ -405,6 +406,7 @@ abstract class TestRunner {
   );
 
   private expectedLimboDocs: DocumentKey[];
+  private expectedInactiveLimboDocs: DocumentKey[];
```

Comment:
Please resolve the ambiguity of the `expectedLimboDocs` name too.

Reply:
Done.